src/Ast.ml

```diff
@@ -1270,19 +1273,19 @@ end = struct
                        | _ -> false )
                | _ -> false ) ->
         true
-    | _ ->
+    | _ -> (
       match ambig_prec (sub_ast ~ctx (Typ typ)) with
       | Some (Some true) -> true
-      | _ -> false
+      | _ -> false )
 
   (** [parenze_cty {ctx; ast}] holds when class type [ast] should be
       parenthesized in context [ctx]. *)
   let parenze_cty ({ctx; ast= cty} as xcty) =
     assert (check_cty xcty ; true) ;
     match xcty with _ ->
-      match ambig_prec (sub_ast ~ctx (Cty cty)) with
+      ( match ambig_prec (sub_ast ~ctx (Cty cty)) with
```

Comment:
Is the open paren here and not at the end of the previous line intentionally?

Reply:
no

Reply:
fixed